src/DataFixtures/ORM/LoadReferentManagedUserData.php

```diff
@@ -25,7 +25,7 @@ public function load(ObjectManager $manager)
             'original_id' => $this->getReference('adherent-1')->getId(),
             'email' => $this->getReference('adherent-1')->getEmailAddress(),
             'postal_code' => $this->getReference('adherent-1')->getPostalCode(),
-            'city' => $this->getReference('adherent-1')->getCity(),
+            'city' => $this->getReference('adherent-1')->getCityName(),
```

Comment:
It is indeed the city name which is currently in the database: https://github.com/EnMarche/worker/blob/master/job_referent_managed_users.go#L32